UI/PauseScreen.cpp

```diff
@@ -78,11 +78,9 @@ void AsyncImageFileView::SetFilename(std::string filename) {
 void AsyncImageFileView::Draw(UIContext &dc) {
 	using namespace Draw;
 	if (!texture_ && !textureFailed_ && !filename_.empty()) {
-		texture_ = dc.GetThin3DContext()->CreateTextureFromFile(filename_.c_str(), DETECT);
+		texture_ = CreateTextureFromFile(dc.GetThin3DContext(), filename_.c_str(), DETECT);
 		if (!texture_)
 			textureFailed_ = true;
-		else if (textureAutoGen_)
-			texture_->AutoGenMipmaps();
```

Comment:
It's a shame that screenshots now look crappy and pixelated again here (#8812) and for savedata.

I realize that any features the now-preferred backend (D3D11) doesn't support will probably be removed, but doesn't D3D11 support mipmap generation?

-[Unknown]